config.yml

```diff
@@ -15,8 +15,8 @@ relaychain:
     - name: charlie
 
 statemine:
-  image: parallelfinance/polkadot:v0.9.9-1
-  chain: statemine-local
+  image: eteissonniere/statemine:polkadot-v0.9.9
```

Comment:
the master is based on `polkadot v0.9.10` @ayushmishra2005

Reply:
can you create a `eteissonniere/statemine:polkadot-v0.9.10`, then I'll tag it to `parallelfinance/statemine:v0.9.10`

Reply:
Raised https://github.com/ETeissonniere/substrate-nodeops/pull/16

Reply:
@ashi20192020ayush I tagged it to parallelfinance/statemine:v0.9.10, can you change it to parallelfinance/statemine:v0.9.10?